lib/acl-checker.js

```diff
@@ -0,0 +1,230 @@
+'use strict'
+
+const async = require('async')
+const path = require('path')
+const url = require('url')
+
+const DEFAULT_ACL_SUFFIX = '.acl'
+
+class ACLChecker {
+  constructor (options = {}) {
+    this.debug = options.debug || console.log
+    this.fetch = options.fetch
+    this.strictOrigin = options.strictOrigin
+    this.suffix = options.suffix || DEFAULT_ACL_SUFFIX
+  }
```

Comment:
We're almost there. There's two pending PRs for rdflib that bring it into conformance to the RDFJS data model. And there's similar progress to rdf-ext on the v1 branch that does the same. and the signature of the `match()` function is at least standardized, and matches.

(There's still work to be done on standardizing the *results* of `match()` - Bergos wants to standardize a DataSet model, whereas we're just returning an array for now. But in any case, I think it's safe to just call `graph.match()` and be reasonably future-proof.)